### ISSUE TITLE
Progress - status should be conveyed via helper text in examples

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11397]
**Describe the enhancement or change**
Examples should be updated to include helper text when status is used. Currently there's nothing conveying the progress status to AT since examples only include visual indication. 

Additionally, this helper text should be linked to the ProgressBar via aria-describedby. This would require editing the Progress code slightly -- either hardcoding the `aria-describedby` on ProgressBar and `id` on ProgressHelperText and/or introducing a couple of new props for consumers to do this themselves.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**
